### PR TITLE
[SPARK-26189][R]Fix unionAll doc in SparkR

### PR DESCRIPTION
**Pull request by huaxingao**
## What changes were proposed in this pull request?

Fix unionAll doc in SparkR

## How was this patch tested?

Manually ran test

### CODE REVIEW

File: R/pkg/R/DataFrame.R [hunk 1]
```diff
@@ -2732,14 +2732,24 @@ setMethod("union",
...
-#' @rdname union
-#' @name unionAll
+#' @param x a SparkDataFrame.
+#' @param y a SparkDataFrame.
+#' @return A SparkDataFrame containing the result of the unionAll operation.
+#' @family SparkDataFrame functions
 #' @aliases unionAll,SparkDataFrame,SparkDataFrame-method
-#' @note unionAll since 1.4.0
+#' @rdname unionAll
+#' @name unionAll
```
Comment (felixcheung) [inline R/pkg/R/DataFrame.R]: I'd add a `@seealso` to union

File: R/pkg/R/DataFrame.R [hunk 2]
```diff
@@ -2732,14 +2732,24 @@ setMethod("union",
...
+#' @examples
+#'\dontrun{
+#' sparkR.session()
+#' df1 <- read.json(path)
+#' df2 <- read.json(path2)
+#' unionAllDF <- unionAll(df1, df2)
+#' }
+#' @note unionAll since 3.0.0
```
Comment (felixcheung) [inline R/pkg/R/DataFrame.R]: this should be `1.4.0` as above

File: R/pkg/R/DataFrame.R [hunk 3]
```diff
@@ -2732,13 +2732,24 @@ setMethod("union",
             dataFrame(unioned)
           })
 
-#' Return a new SparkDataFrame containing the union of rows
-#'
-#' This is an alias for `union`.
+#' Return a new SparkDataFrame containing the union of rows.
+#' This is an alias for \code{union}.
```
Comment (felixcheung) [inline R/pkg/R/DataFrame.R]: actually, we do need a newline - L2735 is the title as this is a new page. could you build the roxygen2 doc to double check?

Comment (huaxingao) [reply R/pkg/R/DataFrame.R]: You are right, it is the title and newline is required. I manually checked.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by felixcheung** [comment]
LGTM

**Comment by felixcheung** [comment]
merged to master

**Pull request closed without merging**